### PR TITLE
feat: add zoom with mouse wheel scroll for 360 images

### PR DESCRIPTION
**Pull request by christjt**
#### Type of change

Feat  

#### Jira ticket :blue_book:

https://cognitedata.atlassian.net/browse/REV-600

## Description :pencil:

Adds zooming capabilities to mouse scroll, and inverts rotation controls.

## Test instructions :information_source:

Using the Viewer.tsx example, load (in 3d-test):

modelId: `4923307033982497`
revisionId: `3100284276970593`

Using the dropdown dat.gui load 360 images with the follow parameters:

- site_id is `helideck-site-2`
- translation offset is  `(11, 49, 32)`
- rotation (defined with axis rotation and  radians) is axis: `(0, 1, 0)` and radians `3.08923`

Then click on one of the 360 image icons to enter the 360 image. Then use the mouse scroll-wheel to zoom.

## Checklist :ballot_box_with_check:

- [x] I am proud of this feature.
- [x] I have performed a self-review of my own code.
- [x] I have added PR type (Feat, Bug, Chore, Test, Docs, Style, Refactor) to the PR title.
- [x] I have manually tested this for different scenarios (different models, formats, devices, browsers).
- [x] I have commented my code in hard-to-understand areas.
- [ ] I have made corresponding changes to the public documentation.
- [ ] I have added unit and visuals tests to prove that my feature works to the best of my ability.
- [x] I have refactored the code for readability to the best of my ability.
- [ ] I have checked that my changes do not introduce regressions in the public documentation.
- [x] I have outlined any known defects / lacking capabilities in the contents of this PR.
- [x] I have listed any remaining work that should follow this PR in the description or jira/miro/etc.
- [x] I have added TSDoc to any public facing changes.
- [ ] I have added "developer documentation" in any package README.md that is related / applicable to this PR.
- [x] I have noted down and am currently tracking any technical debt introduced in this PR.
- [x] I have thoroughly thought about the architecture of this implementation.
- [ ] I have asked peers to test this feature.

### CODE REVIEW

**Comment by Savokr** [comment]
Probably not related to this PR but I noticed that 360 images sprites texture seem to be 1-2px smaller than corresponding canvas graphics objects. It result in "cut" edges at centers of each sprite side. It's noticeable when circle is hovering over bright background (see picture below).


Screenshot 2022-11-25 at 17 27 31

**Review by Savokr — APPROVED** [review]
Looks and feels good, one comment about improving functionality but probably can be done in new PR.

File: viewer/packages/camera-manager/src/StationaryCameraManager.ts [hunk 1]
```diff
@@ -133,10 +146,16 @@ export class StationaryCameraManager implements CameraManager {
...
-    euler.x -= movementY * 0.002;
-    euler.y -= movementX * 0.002;
+    euler.x -= -movementY * 0.0015 * (this._camera.fov / this._defaultFOV);
+    euler.y -= -movementX * 0.0015 * (this._camera.fov / this._defaultFOV);
     euler.x = Math.max(-Math.PI / 2, Math.min(Math.PI / 2, euler.x));
     this._camera.quaternion.setFromEuler(euler);
     this._cameraChangedListener.forEach(cb => cb(this._camera.position, this._camera.position));
   };
+
+  private readonly zoomCamera = (event: WheelEvent) => {
```
[the remark this answers is not in the document]
Comment (christjt) [reply viewer/packages/camera-manager/src/StationaryCameraManager.ts:156]: Think a pragmatic approach would be to define a rotation in the direction of the mouse pointer and when zooming slerp towards that rotation. In any case though, I think we will just leave it as is, and then iterate on this in 4.x.x depending on feedback.

**Comment by christjt** [comment]
> Probably not related to this PR but I noticed that 360 images sprites texture seem to be 1-2px smaller than corresponding canvas graphics objects. It result in "cut" edges at centers of each sprite side. It's noticeable when circle is hovering over bright background (see picture below).
> 
> Screenshot 2022-11-25 at 17 27 31

Created a bug report for this: https://cognitedata.atlassian.net/browse/REV-605

**Pull request merged**